src/isomorphic/constructRoutes.js

```diff
@@ -81,6 +83,9 @@ const pathToActiveWhen = singleSpa[defaultStr]
  */
 export function constructRoutes(routesConfig, htmlLayoutData) {
   if (routesConfig && routesConfig.nodeName) {
+    if (inBrowser && !htmlLayoutData && window.singleSpaLayoutData) {
+      htmlLayoutData = window.singleSpaLayoutData;
```

Comment:
`window.singleSpaLayoutData` is created on the server side, and we check here in the browser if it's defined so that we can reuse the same prop values on both server and browser.